libs/game/textDialogs.ts

```diff
@@ -494,41 +516,48 @@ namespace game {
 
         drawScores() {
             if (this.hasScores) {
+                const scores = this.scores.filter(score => score.value != null);
                 let currY = image.font5.charHeight + 16;
                 if (this.isMultiplayerGame) {
-                    if (this.scores.length === 1) {
-                        // Multiplayer special case: Only one player scored, but it wasn't player 1
-                        const score = this.scores[0];
+                    if (scores.length === 1) {
+                        // Multiplayer special case: Only one player scored
+                        const score = scores[0];
                         score.str = "P" + score.player + ":" + score.value;
                         this.image.printCenter(
                             score.str,
                             currY,
                             screenColor(1),
                             image.font5
                         );
+                        if (score.winner) {
```

Comment:
I probably should have thought of this before but for multiplayer, what should we do if the two 'best' players have the same score? Doesn't really matter for this new block as we're explicitly passing in the winner but I guess we probably need a 'these 2 / 3 / all players tied' message in case of ties? Fine to pass this off / wouldn't be part of this pr anyways

Reply:
Ties are not supported at this point. Will file an issue to track it.